pkg/pgmodel/ingestor/series_writer.go

```diff
@@ -212,7 +214,7 @@ func (h *seriesWriter) fillLabelIDs(ctx context.Context, infos map[string]*perMe
 	infoBatches := make([]*perMetricInfo, 0)
 	items := 0
 	for metricName, info := range infos {
-		names, values := info.labelList.Get()
+		names, values := info.labelsToGetOrCreate.Get()
```

Comment:
This looks weird. The `.Get()` of `..GetOrCreate` returns `names` and `values`? I think the function needs renaming.

Reply:
True. Renamed to `labelsToFetch` - this is essentially list of labels we haven't found in our cache and need to fetch from DB